docker/json-server/Dockerfile

```diff
@@ -1,18 +1,3 @@
-## Build front end assets so json-server can route to them
-FROM node:14.17.5-alpine3.14 as build-assets
-
-WORKDIR /app
-
-RUN mkdir -p web/static
-
-COPY web/assets web/assets
-COPY webpack.config.js .
-COPY package.json .
-COPY yarn.lock .
-
-RUN yarn install
-RUN yarn build
-
```

Comment:
for my own understanding - do we not longer need these? Have we set these up somewhere else in the repo?

Reply:
Before we introduced Mux to do more advanced routing, this was necessary as the web assets were just placed on the file system in the public directory, so in order to serve them with json-server running, we had to make them available in its container. This meant duplicating the asset build process, which increased build time and complexity (albeit only locally and not by much).

Now that we have Mux and its static file routing, the app container can serve its own assets, so this part is no longer necessary.